worker/containerd/containerd.go

```diff
@@ -80,13 +70,13 @@ func newContainerd(root string, client *containerd.Client, snapshotterName, ns s
 	if apparmorProfile != "" {
 		xlabels[wlabel.ApparmorProfile] = apparmorProfile
 	}
-	xlabels[wlabel.ContainerdNamespace] = ns
+	xlabels[wlabel.ContainerdNamespace] = client.DefaultNamespace()
```

Comment:
It doesn't look like containerd guarantees that `DefaultNamespace()` returns a non-zero value. In that case we need to validate it in this function before using.